src/main/java/flight/reservation/plane/Helicopter.java

```diff
@@ -1,25 +1,21 @@
 package flight.reservation.plane;
 
-public class Helicopter {
-    private final String model;
-    private final int passengerCapacity;
+public class Helicopter extends Aircraft {
+    private final int H1_PASSENGER_CAPACITY = 4;
+    private final int H2_PASSENGER_CAPACITY = 6;
+    private final int CREW_PASSENGER_CAPACITY = 2;
+    private final String H1_MODEL_IDENTIFIER = "H1";
+    private final String H2_MODEL_IDENTIFIER = "H2";
```

Comment:
Would be nicer if the capacities and identifiers would be grouped in some kind of enum/struct.